OSDOCS#7562: Scheduling workloads on multi-arch clusters (#63915)
openshift/openshift-docs: modules/multi-architecture-scheduling-overview.adoc

// module included in the following assembly 
//
//post_installation_configuration/configuring-multi-arch-compute-machines/multi-architecture-compute-managing.adoc

:_content-type: CONCEPT
[id="multi-architecture-scheduling-overview_{context}"]
= Scheduling workloads on clusters with multi-architecture compute machines 

> Before deploying a workload onto a cluster with compute nodes of different architectures, you must configure your compute node scheduling process so the pods in your cluster are correctly assigned. 

Reviewer: Nice!

> There are various ways you can schedule workloads onto multi-architecture nodes for your cluster. You can use a node affinity, a node selector or a `RuntimeClass` object to select the node you want the pod to schedule onto. A `RuntimeClass` configuration also allows you to use scheduling mechanisms, like taints and tolderations. 

Reviewer 2: should we remove the RuntimeClass bits here?

> There are various ways you can schedule workloads onto multi-architecture nodes for your cluster. You can use a node affinity or a node selector to select the node you want the pod to schedule onto. You can also use scheduling mechanisms, like taints and tolderations, when using node affinity or node selector to correctly schedule workloads. 

Reviewer: ```suggestion
You can schedule workloads onto multi-architecture nodes for your cluster in serveral ways. For example, you can use a node affinity or a node selector to select the node you want the pod to schedule onto. You can also use scheduling mechanisms, like taints and tolderations, when using node affinity or node selector to correctly schedule workloads. 
```
